packages/desktop-client/src/components/modals/TrackingBudgetMenuModal.tsx

```diff
@@ -86,11 +79,9 @@ export function TrackingBudgetMenuModal({
             </Text>
             <FocusableAmountInput
               value={integerToAmount(budgeted || 0)}
-              focused={amountFocused}
-              onFocus={() => setAmountFocused(true)}
-              onBlur={() => setAmountFocused(false)}
               onEnter={close}
               zeroSign="+"
+              defaultFocused={true}
```

Comment:
React inputs have `autoFocus` which automatically focuses the element on render. Maybe we can remove this `defaultFocused` and just use the built in `autoFocus` prop

Reply:
`FocusableAmountInput` needs control over `focused` state, i.e. to display little button to toggle positive/negative input.